msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -349,23 +425,27 @@ An array of `string` which specifies which permissions the app requests, which l
 * `identity` &emsp; Requires user identity information
 * `messageTeamMembers` &emsp; Requires permission to send direct messages to team members
 
+Changing these permissions when updating your app will cause your users to repeat the consent process the first time they run the updated app. See [Updating your app](~/publishing/apps-publish.md#updating-your-app) for more information.
```

Comment:
Im pretty sure the permissions defined in manifest are not used anywhere and this statement isnt true

Reply:
Do you know who the right PM to talk to about this is?

Reply:
@billbliss